loader_hub/firestore/base.py

```diff
@@ -0,0 +1,70 @@
+"""Firestore Reader."""
+
+from typing import Any, List, Optional
+from google.cloud import firestore
```

Comment:
due to the way our CI/imports are setup atm, let's move this as a lazy import within the function that its called